pkg/hostagent/event_poster.go

```diff
@@ -113,9 +113,9 @@ func (agent *HostAgent) processPacketEvent(packetEvent PacketEvent, currTime tim
 	agent.indexMutex.Lock()
 	srcPodKey, srcOk := agent.podIpToName[packetEvent.SourceIP]
 	agent.indexMutex.Unlock()
-	agent.log.Debug("srcPodKey for ", packetEvent.SourceIP, ": ", srcPodKey)
+	agent.log.Trace("srcPodKey for ", packetEvent.SourceIP, ": ", srcPodKey)
```

Comment:
should this be below in the else...it doesn't seem useful if !srcOk

Reply:
Agreed. Will change.